modules/terraform/init_test.go

```diff
@@ -35,3 +36,36 @@ func TestInitBackendConfig(t *testing.T) {
 
 	assert.FileExists(t, remoteStateFile)
 }
+
+func TestInitPluginDir(t *testing.T) {
+	t.Parallel()
+
+	pluginDir, err := ioutil.TempDir("", t.Name())
+	require.NoError(t, err)
+
+	terraformFixture := "../../test/fixtures/terraform-basic-configuration"
+
+	initializePluginsFolder, err := files.CopyTerraformFolderToTemp(terraformFixture, t.Name())
+	require.NoError(t, err)
+
+	testFolder, err := files.CopyTerraformFolderToTemp(terraformFixture, t.Name())
+	require.NoError(t, err)
```

Comment:
NIT: add `defer os.RemoveAll(testFolder)` calls so we clean up these temp folders.

Reply:
fixed!